### PR TITLE
Removing Sinc, Adding Dissolv

### PR DESCRIPTION
**Pull request by Liam2263**
Dissolv is v2 of Sinc (C# RedUtils)
If you need the un-compiled version of it then I will gladly send you it in discord dms.

Allow edits by maintainers is on btw.

If you catch any un-needed files please let me know!

Was previously tested before the making of this pr.

### CODE REVIEW

**Comment by VirxEC** [comment]
you do have a \_\_pycache\_\_ folder and some unneeded .pdb files

**Comment by Liam2263** [comment]
I have removed those files now,

Thanks!

**Review by VirxEC — APPROVED** [review]

**Comment by Liam2263** [comment]
I’m going to close this pr due to the fact I want to submit another bot to the botpack with Dissolv and that might be a while or so, I don’t want to send another pr for another bot like 3 days after this one gets approved when it does.

I should be sending another pr right before Triple Threat.

Thanks!

**Pull request closed without merging**
